de/events.json

```diff
@@ -1 +1,27 @@
-{}
+{
+  "page-title": "Nächste Events ({{platform}}) - Eric's GoW Tools",
+  "page-desc": "Liste von zukünftigen Events in Gems of War, täglich aktualisiert.",
+
+  "head-name": "Name",
+  "head-start-date": "Startdati,",
+  "head-end-date": "Enddatum",
+  "head-duration": "Dauer",
+  "head-troop-pet-class": "Truppe/Pet/Klasse",
+  "head-kingdom": "Königreich",
+  "head-currency": "Währung",
+  "head-medal": "Orden",
+
+  "event-duration": "{{count}} Tag",
+  "event-duration_plural": "{{count}} Tage",
+
+  "event-guildwars": "Gildenkrieg",
+  "event-bounty": "Bounty",
```

Comment:
Use the german term "Kopfgeld" here, this term is also used on all bounty troops.